### ISSUE TITLE
Add sources for SCCS data

### ISSUE THREAD

**Issue by xrotwang** [D-PLACE/dplace-data#129]
SCCS data uses the same sources per society as EA, so the `sources` field in `data.csv` should be filled in accordingly from an appropriate variable in EA, e.g. `Subsistence economy: hunting`.

**Comment by kirbykat** [comment]
Yes, for now sources are general across the majority of the variables in EA, and all variables in SCCS. I'm hoping an RA can eventually tackle links between specific variables and references.
